lib/Targeting/Code/TargetingCodeGenerator.php

```diff
@@ -17,7 +17,7 @@
 
 namespace Pimcore\Targeting\Code;
 
-use Pimcore\Analytics\Code\CodeBlock;
+use Pimcore\Bundle\GoogleMarketingBundle\Code\CodeBlock;
```

Comment:
the file is in `lib/Targeting/Code/TargetingCodeGenerator.php`, which may hint that this won't work if the Marketing Bundle is *not* installed, need some checks or getting moved, especially in the `TargetingListener`
see also https://github.com/pimcore/pimcore/blob/5fa71a342b8f070617a3f848d2a5365e3e3c2f7c/lib/Event/Targeting/TargetingCodeEvent.php#L20

I see that CodeBlock got moved but the former one in still in the branch
https://github.com/pimcore/pimcore/blob/5fa71a342b8f070617a3f848d2a5365e3e3c2f7c/lib/Analytics/Code/CodeBlock.php#L18-L24

Reply:
The TargetingCodeGenerator now moved to the PersonalizationBundle
The Codeblock from the lib will be moved to both bundles PersonalizationBundle and GoogleMarketingBundle.
See #14398